spring-kafka/src/main/java/org/springframework/kafka/retrytopic/DestinationTopicContainer.java

```diff
@@ -65,9 +66,10 @@ public interface DestinationTopicContainer {
 	 * DLT for the given topic, or null if none is found.
 	 * @param mainListenerId the listener id.
 	 * @param topicName the topic name for which to look the DLT for
+	 * @param exc the exception which is being handled
 	 * @return The {@link DestinationTopic} instance corresponding to the DLT.
 	 */
 	@Nullable
-	DestinationTopic getDltFor(String mainListenerId, String topicName);
+	DestinationTopic getDltFor(String mainListenerId, String topicName, Exception exc);
```

Comment:
Cannot we have another method to avoid a breaking change?
It also would be great to have it as a `default` to avoid a immediate upgrade compilation problem.
We may have the current method deprecated then.